java/src/main/java/de/gdata/vaas/Vaas.java

```
@@ -651,4 +652,13 @@ private void throwVaasException(ExecutionException e) throws VaasClientException
     private static <E extends Throwable> void throwAsUnchecked(Exception exception) throws E {
         throw (E) exception;
     }
+
+    @Override
+    public void close() throws IOException {
+        try {
+            this.disconnect();
+        } catch (InterruptedException e) {
```

Catch all exceptions here